### PR TITLE
pkg/*: put $(MAKE) in quotes in all packages

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

This PR is a cosmetic improvement of some package build output. Some `make` are not surrounded by double quotes and this shifts the corresponding build line from regular RIOT modules.

### Issues/PRs references

None

### CODE REVIEW

**Comment by aabadie** [comment]
nobody interested ? @cladmi  ?

File: pkg/cmsis-dsp/Makefile [hunk 1]
```diff
@@ -8,6 +8,6 @@ CFLAGS += -Wno-strict-aliasing
 .PHONY: all
 
 all: git-download
-	$(MAKE) -C $(PKG_BUILDDIR)
+	"$(MAKE)" -C $(PKG_BUILDDIR)
```
Comment (bergzand) [inline pkg/cmsis-dsp/Makefile]: While not necessarily related to this PR, I suspect the `$(PKG_BUILDDIR)` variables should also be quoted here and in the other makefiles. This looks like it breaks when there is whitespace in `PKG_BUILDDIR`.

**Comment by kaspar030** [comment]
Feel free to change the PR and commit title to "put $(MAKE) in quotes in packages", as for output consistency, the quite flags would need to be adapted...

**Comment by aabadie** [comment]
@kaspar030, commit message and PR title are now updated with your suggestion. Thanks !

**Comment by kaspar030** [comment]
Let's go!

File: pkg/cmsis-dsp/Makefile (hunk 1 above)
Comment (kaspar030) [reply pkg/cmsis-dsp/Makefile]: True... We could open an issue and tag it for newbies?

**Pull request merged**